.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
```

Comment:
I think you have added this file by mistake

Reply:
oh right ! thanks for pointing that out